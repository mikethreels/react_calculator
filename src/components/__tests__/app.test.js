import React from 'react';
import renderer from 'react-test-renderer';
import App from '../app';

describe('App', () => {
  it('Tests if component renders correctly', () => {
    const tree = renderer
      .create(<App />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
