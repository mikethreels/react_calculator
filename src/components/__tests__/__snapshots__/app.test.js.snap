// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Tests if component renders correctly 1`] = `
<div>
  <div
    className="display"
  >
    0
  </div>
  <div
    className="panel"
  >
    <div
      className="group1"
    >
      <button
        className="buttongroup1"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="buttongroup1"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="buttongroup1"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="buttongroup1last"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
    </div>
    <div
      className="group2"
    >
      <button
        className="buttongroup2"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="buttongroup2"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="buttongroup2"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="buttongroup2last"
        onClick={[Function]}
        type="button"
      >
        X
      </button>
    </div>
    <div
      className="group3"
    >
      <button
        className="buttongroup3"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="buttongroup3"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="buttongroup3"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="buttongroup3last"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
    </div>
    <div
      className="group4"
    >
      <button
        className="buttongroup4"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="buttongroup4"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="buttongroup4"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="buttongroup4last"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
    </div>
    <div
      className="group5"
    >
      <button
        className="buttongroup5first"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="buttongroup5"
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="buttongroup5last"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
